### PR TITLE
fix pylint error

### PR DESCRIPTION
**Pull request by MarWeUMR**
Attempt ([b1450d6](https://github.com/geo-engine/geoengine-python/commit/b1450d653181f1754a251ee040b3c5952cdbf3af)) to fix the errors from pylint about missing timeout values. The provided values are initial best guesses.

### CODE REVIEW

**Comment by MarWeUMR** [comment]
[requests.post() und requests.get() W3101 Fehler](https://trello.com/c/FIYZTY13/962-requestspost-und-requestsget-w3101-fehler)

File: geoengine/workflow.py [hunk 1]
```diff
@@ -64,6 +64,7 @@ class Workflow:
 
     __workflow_id: WorkflowId
     __result_descriptor: ResultDescriptor
+    __timeout: int = 10
```
Comment (ChristianBeilschmidt) [inline geoengine/workflow.py:67]: Some methods in `workflows.py` use this variable and others have it in their params with a default value.
I would prefer the second method here. Moreover, maybe change 10 to 60 seconds. We just don't want to wait forever.

File: geoengine/auth.py [hunk 2]
```diff
@@ -34,6 +34,7 @@ class Session:
     __id: UUID
     __valid_until: Optional[str] = None
     __server_url: str
+    __timeout: int = 10
```
Comment (ChristianBeilschmidt) [inline geoengine/auth.py:37]:
```suggestion
    __timeout: int = 60
```

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: geoengine/auth.py (hunk 2 above)
Comment (MarWeUMR) [reply geoengine/auth.py]: [5db5bbf](https://github.com/geo-engine/geoengine-python/pull/71/commits/5db5bbf74e6c3b66993b3381ad4d3815d4cda9fb)

File: geoengine/workflow.py (hunk 1 above)
Comment (MarWeUMR) [reply geoengine/workflow.py]: [12478ac](https://github.com/geo-engine/geoengine-python/pull/71/commits/12478ac297e420b824d9393ef19c88dcd97b5e3b)

Removed the field and made it a parameter for all methods with 60s default.

**Review by ChristianBeilschmidt — DISMISSED** [review]

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**